partition-raw-ais/src/main/scala/uk/gov/ukho/ais/partitioning/PartitionRawAis.scala

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ukho.ais.partitioning
+
+import uk.gov.ukho.ais.{Schema, Session}
+import uk.gov.ukho.ais.partitioning.RddWriters.YearMonthPartitionWriter
+
+class PartitionRawAis {
+  private val YEAR_EXPRESSION = "year(acquisition_time) as year"
+  private val MONTH_EXPRESSION = "month(acquisition_time) as month"
+
+  def main(args: Array[String]): Unit = {
+    Session.init("Partition Raw AIS")
+
+    implicit val config: Config = ConfigParser.parse(args)
+
+    val selectClause: Seq[String] = Seq(
+      Schema.AIS_SCHEMA.map(field => field.name): _*) ++
```

Comment:
Surely this can be unwrapped?

```suggestion
      Schema.AIS_SCHEMA.map(field => field.name) ++ Seq(YEAR_EXPRESSION, MONTH_EXPRESSION)
```